Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataPatientBean.java

```diff
@@ -83,82 +113,246 @@ public void setSelectedPatient(Patient selectedPatient) {
         this.selectedPatient = selectedPatient;
     }
 
-    // DATABASE-METHODS-getAll, delete, save
+    // selected-record
+    public Personname getSelectedPersonname() {
+        return selectedPersonname;
+    }
+
+    public void setSelectedPersonname(Personname selectedPersonname) {
+        this.selectedPersonname = selectedPersonname;
+    }
+
+    public Identifier getSelectedIdentifier() {
+        return selectedIdentifier;
+    }
+
+    public void setSelectedIdentifier(Identifier selectedIdentifier) {
+        this.selectedIdentifier = selectedIdentifier;
+    }
+
+    public Address getSelectedAddress() {
+        return selectedAddress;
+    }
+
+    public void setSelectedAddress(Address selectedAddress) {
+        this.selectedAddress = selectedAddress;
+    }
+
+    public Phonenumber getSelectedPhonenumber() {
+        return selectedPhonenumber;
+    }
+
+    public void setSelectedPhonenumber(Phonenumber selectedPhonenumber) {
+        this.selectedPhonenumber = selectedPhonenumber;
+    }
+
+    // database-methods
+    public List<Phonenumber> getPhonenumbers() {
+        if (isValidPatientId()) {
+            return loadTestDataService.getAllPhonenumbersBy(getPatientId());
+        } else {
+            return new ArrayList<>();
+        }
+    }
+
+    public boolean deletePhonenumber() {
+        if (selectedPhonenumber != null) {
+            return loadTestDataService.deletePhonenumber(selectedPhonenumber);
+        } else {
+            addPatientErrorMessages("Phonenumber have to be select to be deleted");
+        }
+        return false;
+    }
+
+    public boolean savePhonenumber() {
+        boolean actionResult = false;
+
+        if (isValidPatientId()) {
+            try {
+                Phonenumber phonenumber = readPhonenumber();
+                phonenumber.setPatient(withPatient);
+                actionResult = loadTestDataService.savePhonenumber(phonenumber);
+
+                if (actionResult) {
+                    addPatientInfoMessages("Save Phonenumber Successful: " + phonenumber.getPhonenumberId());
+                    clearInputsPhonenumbers();
+                }
+            } catch (LoadTestDataException e) {
+                logPatientError("Patient Phonenumber", e);
+            }
+        } else {
+            addPatientErrorMessages("Phonenumber cannot be save: patient does not exist.");
+        }
+        return actionResult;
+    }
+
+    public List<Address> getAddresses() {
+        if (isValidPatientId()) {
```

Comment:
A method should have only one return statement instead of two. Use a local variable to get the result and return the local variable.